firebase-firestore/src/main/java/com/google/firebase/firestore/model/FieldIndex.java

```diff
@@ -141,7 +141,7 @@ public static IndexOffset create(
     /**
      * Creates an offset that matches all documents with a read time higher than {@code readTime}.
      */
-    public static IndexOffset create(SnapshotVersion readTime) {
+    public static IndexOffset create(SnapshotVersion readTime, int largestBatchId) {
```

Comment:
optional: rename to avoid unintentional usage where caller expects offset to be the exact provided readtime.

some quick examples: createForNextReadTime/createSucessor?

Reply:
createSucessor() it is.